pages/index.js

```diff
@@ -2,11 +2,22 @@ import Heading from "@/components/Heading";
 import EmotionForm from "@/components/EmotionForm";
 import EmotionList from "@/components/EmotionList";
 
-export default function HomePage({ objects, onCreateEmotion }) {
+export default function HomePage({
+  objects,
+  onCreateEmotion,
+  onUpdateEmotion,
+}) {
   return (
     <div>
       <Heading>Emotion Tracker</Heading>
-      <EmotionForm onCreateEmotion={onCreateEmotion} />
+      <EmotionForm
+        objects={objects}
+        onCreateEmotion={onCreateEmotion}
+        onUpdateEmotion={onUpdateEmotion}
+        onSubmit={(data) => {
+          onCreateEmotion({ ...data });
+        }}
```

Comment:
The equivalent of what I wrote above applies here. But please think about it again, perhaps I'm overlooking something.

Reply:
see above